src/models/userModel.ts

```diff
@@ -1,18 +1,23 @@
 import mongoose, { Schema } from "mongoose";
 import UserType from "../interfaces/userType";
 
+
+const TOKEN_SECRET = String(process.env.TOKEN_SECRET);
 const userSchema = new Schema<UserType>(
     {
         firstName: { type: String, required: true },
         lastName: { type: String, required: true },
         email: { type: String, required: true },
         mobile: { type: Number, required: true },
-        password: { type: String, required: true },
+        password: { type: String, required: true},
         active: { type: Boolean, required: true },
-        verificationCode: { type: String, required: true, unique: true },
+        verificationCode: { type: String, required: true, unique: true, select:false },
```

Comment:
This should not be unique and should not be required

Reply:
Removed both required and unique field